cli/commands/deprecated.go

```diff
@@ -25,61 +23,127 @@ const (
 	CommandOutputAllName   = "output-all"
 	CommandValidateAllName = "validate-all"
 
-	CommandHCLFmtName         = "hclfmt"
-	CommandHCLValidateName    = "hclvalidate"
-	CommandValidateInputsName = "validate-inputs"
+	CommandRunAllName             = "run-all"
```

Comment:
Looks like CommandRunAllName is unused
```
$ grep -irn "CommandRunAllName" .
./cli/commands/deprecated.go:26:        CommandRunAllName             = "run-all"
```

and a couple of others

```
CommandRunAllName
CommandGraphName
CommandTerragruntInfoName
CommandOutputModuleGroupsName
CommandGraphDependenciesName
```

Reply:
Since I cut all these PRs from the final code where all commands were deprecated, I forgot to remove them, they are used in other PRs. If I remove them from this branch now, I'll have to insert them into other branches, but since they will be merged at the same time, it makes sense to turn a blind eye to it.